wp-admin/js/widgets/media-image-widget.js

```diff
@@ -187,23 +187,6 @@
 
 			mediaFrame.open();
 
-			/*
-			 * Make sure focus is set inside of modal so that hitting Esc will close
-			 * the modal and not inadvertently cause the widget to collapse in the
-			 * customizer.
-			 */
-			mediaFrameContentView = mediaFrame.views.get( '.media-frame-content' )[0];
-			if ( mediaFrameContentView.model.dfd ) {
-				mediaFrameContentView.model.dfd.done( function() {
-					_.defer( function() { // Next tick.
-						mediaFrameContentView.$el.find( '[data-setting="caption"]:first' ).focus();
-					} );
-				} );
-			} else {
-				_.defer( function() {
-					jQuery( '.media-frame-content [data-setting="caption"]:first' ).focus();
-				} );
-			}
```

Comment:
@adamsilverstein do you think that the improvements to media modal focusing will be made to the underlying media JS? Do you concur that there seems to be a problem with the focus behavior when opening the modal?